apps/remix-ide/src/app/tabs/compileTab/compilerContainer.js

```diff
@@ -174,25 +174,35 @@ class CompilerContainer {
 
   render () {
     this.compileTabLogic.compiler.event.register('compilerLoaded', (version) => this.setVersionText(version))
-    this.fetchAllVersion((allversions, selectedVersion, isURL) => {
+    this.fetchAllVersion((allversions, selectedVersion) => {
```

Comment:
`isURL` ? I think this shouldn't be part of this PR. This was added during a previous PR by @Aniket-Engg